packages/vaex-ml/vaex/ml/linear_model.py

```diff
@@ -66,8 +65,7 @@ def fit(self, dataset, y_expression, progress=False):
             def coordinates(expression, limits, shape):
                 if dataset.is_category(expression):
                     return np.arange(dataset.category_count(expression))
-                else:
-                    return dataset.bin_centers(expression, limits, shape)
+                return dataset.bin_centers(expression, limits, shape)
```

Comment:
matter of taste, some would convert this back :)

Reply:
Personally, I think that if syntax should be simplified in Early Return Pattern. Related comments are in this link.

https://www.itamarweiss.com/personal/2018/02/28/return-early-pattern.html